### PR TITLE
update docs simple formatter widget

### PR DESCRIPTION
**Pull request by hug963**
## Subject

Update the docs to document how to set ckeditor_context on a sonata_simple_formatter_type form field.

### CODE REVIEW

File: Resources/doc/reference/formatter_widget.rst [hunk 1]
```diff
@@ -17,7 +17,8 @@ The ``sonata_simple_formatter_type_selector`` widget has been implemented to all
     <?php
 
     $formMapper->add('comment', 'sonata_simple_formatter_type', array(
-        'format' => 'markdown'
+        'format' => 'markdown',
+        'ckeditor_context' => 'default', // Optional
```
Comment (OskarStark) [inline Resources/doc/reference/formatter_widget.rst]: Optional -> optional

**Comment by Soullivaneuh** [comment]
Doc looks good.

I let another contributor using this bundle merge this PR after validation.

cc @sonata-project/contributors

**Comment by OskarStark** [comment]
@core23 can you please review this?

**Comment by core23** [comment]
Thank you @hug963

**Pull request merged**